extra/poetry/poetry_build.sh

```diff
@@ -11,7 +11,7 @@ cd build
 ln -s ../dist .
 
 mkdir nanopb
-tar xf tmp.tar README.md generator
+tar xf tmp.tar README.md generator *.c *.h
```

Comment:
should we move the C source files under their own directory inside the pip package, e.g., under `src`?

Reply:
Yes, these should exist under the main nanopb package so we can use `importlib.resources` to access them.